### PR TITLE
Remove personal pronouns

### LINKED ISSUE #115: Removal of Personal Pronouns From Specs
## Introduction
The specifications have a mix of formal and informal language. Use of personal pronouns is generally common in informal text, and generally avoided in formal text.

It has been identified that:

* [Informal language exists and should be changed](https://github.com/status-im/specs/pull/114#discussion_r424382541)
* [Changes to the language would constitute a significant change to the specs](https://github.com/status-im/specs/pull/114#issuecomment-627979770)
* [Change should be made in a dedicated PR](https://github.com/status-im/specs/pull/114#issuecomment-628747555)

> The other node types are optional, but **it is** RECOMMEND **to** implement a `Mailserver`

## Proposal

* [x] Decide if specifications should have a consistent tone and mode of language (**Answer: Yes**)
* [x] If a consistent tone is required, what the tone should be (**Formal tone**)
* [x] Implement the changes in language

## Notes
These changes could also have a major impact on the structure of the specification documents. For example:

[1-client.md#p2p-overlay-1](https://github.com/status-im/specs/blob/master/docs/stable/1-client.md#p2p-overlay-1) has the following subsections

* Why devp2p? Why not use libp2p?
* What about other RLPx subprotocols like LES, and Swarm?
* Why do you use Whisper?
* I heard you were moving away from Whisper?
* Why do you not use Discovery v5 for node discovery?
* I heard something about mailservers being trusted somehow?

These sections are more approachable and easier to read as they are written in an inviting and engaging tone. I feel that **we should make a distinction between what is supplementary information that can be written in an informal manner and what is instructive or descriptive information that is better suited to formal language.**

## References

* https://www.uts.edu.au/current-students/support/helps/self-help-resources/grammar/formal-and-informal-language

### PR DESCRIPTION
**Pull request by Samyoul**
Resolves #115 

## Why make the change?

Technical documents should be written in formal unambiguous language, much of the specifications use a mixture of language modes.

## What has changed

- Removed use of personal pronouns
- Removed inappropriate use of [passive voice](https://en.wikipedia.org/wiki/English_passive_voice)
- Removed ambiguous subjects (mostly a consequence of removing passive voice)

### CODE REVIEW

**Comment by Samyoul** [comment]
For details on the changes see [the addition to the README](https://github.com/status-im/specs/tree/update/personal-pronouns#language-mode)

File: README.md [hunk 1]
```diff
@@ -39,6 +39,67 @@ npm install
 npm run lint
 ```
 
+### Language mode
+
+- Specifications SHOULD use formal technical language (*different from academic language*).
+- Were appropriate, language SHOULD NOT use personal pronouns.
```
Comment (decanus) [inline README.md]:
```suggestion
- Where appropriate, language SHOULD NOT use personal pronouns.
```

**Review by decanus — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Samyoul) [reply README.md]: Ah, thank you. Good catch.

File: docs/draft/3-whisper-usage.md [hunk 2]
```diff
@@ -164,9 +162,9 @@ for i = 0; i < topicLen; i++ {
 
 Whisper is broadcast-based protocol. In theory, everyone could communicate using a single topic but that would be extremely inefficient. Opposite would be using a unique topic for each conversation, however, this brings privacy concerns because it would be much easier to detect whether and when two parties have an active conversation.
 
-Partitioned topics are used to broadcast private messages efficiently. By selecting a number of topic, it is possible to balance efficiency and privacy.
+Nodes use Partitioned topics to broadcast private messages efficiently. By selecting a number of topic, it is possible to balance efficiency and privacy.
```
Comment (decanus) [inline docs/draft/3-whisper-usage.md]:
```suggestion
Nodes use partitioned topics to broadcast private messages efficiently. By selecting a number of topic, it is possible to balance efficiency and privacy.
```

File: docs/draft/3-whisper-usage.md [hunk 3]
```diff
@@ -277,23 +275,23 @@ To exchange messages with client B, a client A SHOULD:
 - Listen to client's B Contact Code Topic to retrieve their bundle information, including a list of active devices
 - Send a message on client's B partitioned topic
 - Listen to the Negotiated Topic between A & B
-- Once a message is received from B, the Negotiated Topic SHOULD be used
+- Once client A receives a message from B, the Negotiated Topic SHOULD be used
```
Comment (decanus) [inline docs/draft/3-whisper-usage.md]:
```suggestion
- Once client `A` receives a message from `B`, the Negotiated Topic SHOULD be used
```

**Review by decanus — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -39,6 +39,67 @@ npm install
 npm run lint
 ```
 
+### Language mode
```
Comment (oskarth) [inline README.md]: This part of the README is getting quite long... can we pull it out into its own STYLE/GUIDELINE md file? And include other related headers. Make README more scannable and friendly

File: docs/draft/3-whisper-usage.md [hunk 5]
```diff
@@ -58,7 +56,7 @@ encryption properties to support asynchronous chat.
 * *Whisper network*: a group of Whisper nodes connected together through the internet connection and forming a graph
 * *Message*: decrypted Whisper message
```
Comment (oskarth) [inline docs/draft/3-whisper-usage.md]: missing

**Review by oskarth — APPROVED** [review]
Thanks! The guide looks to me. I didn't review all the changes in detail since there are quite a lot, but in general it looks good. I trust you followed your own guidelines :P

File: docs/draft/3-whisper-usage.md (hunk 5 above)
Comment (Samyoul) [reply docs/draft/3-whisper-usage.md]: Hey @oskarth, I'm sorry, I don't understand what is missing. Could you give me some more details? Thank you.

Comment (oskarth) [reply docs/draft/3-whisper-usage.md]: a decrypted

**Comment by Samyoul** [comment]
Hey @decanus , I've done some tidying up, and I've addressed the changes you've suggested. Would you mind giving this PR another look over?

Note I've addressed something that Oskar asked me to do https://github.com/status-im/specs/pull/132#discussion_r433167487, which introduces a new `STYLE-GUIDELINE.md` for all things related to how writing should be done.

**Comment by decanus** [comment]
@Samyoul I'll have another look tomorrow.

File: docs/draft/12-sticker-pack.md [hunk 6]
```diff
@@ -41,25 +41,25 @@ The minimum sticker image resolution is 512x512, and its background SHOULD be tr
...
 
 ### Security
 The IPFS gateway acts as an end-user of the IPFS and allows users of the gateway to access IPFS without connection to the P2P network.
 Usage of a gateway introduces potential risk for the users of that gateway provider. In case of a compromise in the security of the provider, meta information such as IP address, User-Agent and other of its users can be leaked.
-If the provider servers are unavailable the access trough gateway to the IPFS network is lost.
+If the provider servers are unavailable the node loses access through the gateway to the IPFS network.
 
 ### Status sticker usage
-When a sticker is shown in the app, Status app makes an http GET request to IPFS gateway using the hyperlink. 
+When the app shows a sticker, Status app makes an http GET request to IPFS gateway using the hyperlink.
```
Comment (kdeme) [inline docs/draft/12-sticker-pack.md]:
```suggestion
When the app shows a sticker, the Status app makes an HTTP GET request to IPFS gateway using the hyperlink. 
```

File: docs/draft/13-3rd-party.md [hunk 7]
```diff
@@ -61,7 +61,7 @@ Infura hosts a collection of full nodes for the Ethereum network and provides an
 Status works on mobile devices and therefore can't rely on local node. So all communication to Ethereum network happens via Infura.
 
 ##### Concerns
-Making http request means that a user leaks metadata, which can be used in various attacks if the service is hacked.
+Making http request means that a user leaks metadata, which can be used in various attacks if an attacker hacks the service.
```
Comment (kdeme) [inline docs/draft/13-3rd-party.md]:
```suggestion
Making a HTTP request means that a user leaks metadata, which can be used in various attacks if an attacker hacks the service.
```

File: docs/draft/13-3rd-party.md [hunk 8]
```diff
@@ -84,7 +84,7 @@ CryptoCompare is a service that shows live streaming prices, charts and analysis
 Status regularly fetches crypto prices from CryptoCompare. Using that info Status calculates fiat value for transaction or wallet assets.
 
 ##### Concerns
-Making http request means that a user leaks metadata, which can be used in various attacks if the service is hacked.
+Making http request means that a user leaks metadata, which can be used in various attacks if an attacker hacks the service.
```
Comment (kdeme) [inline docs/draft/13-3rd-party.md]:
```suggestion
Making a HTTP request means that a user leaks metadata, which can be used in various attacks if an attacker hacks the service.
```

File: docs/draft/13-3rd-party.md [hunk 9]
```diff
@@ -97,7 +97,7 @@ There is a set of services that used for getting information about collectibles:
 
 
 ##### Concerns
-Making http request means that a user leaks metadata, which can be used in various attacks if the service is hacked.
+Making http request means that a user leaks metadata, which can be used in various attacks if an attacker hacks the service.
```
Comment (kdeme) [inline docs/draft/13-3rd-party.md]:
```suggestion
Making a HTTP request means that a user leaks metadata, which can be used in various attacks if an attacker hacks the service.
```

File: docs/stable/11-waku-mailserver.md [hunk 10]
```diff
@@ -70,7 +70,7 @@ The envelope MUST be encrypted with a symmetric key agreed between the requester
 
 Historic messages MUST be sent to a peer as a packet with a P2P Message code (`0x7f`) followed by an array of Waku envelopes.
 
-In order to receive historic messages from a `Mailserver`, a node MUST trust the selected `Mailserver`, that is allow to receive packets with the P2P Message code. By default, such packets are discarded.
+In order to receive historic messages from a `Mailserver`, a node MUST trust the selected `Mailserver`, that is allowed to receive packets with the P2P Message code. By default, the node discards such packets.
```
Comment (kdeme) [inline docs/stable/11-waku-mailserver.md]: I think the original text here was probably supposed to be something like: "that is **to** allow to receive packets with the P2P Message code"

So perhaps:
```suggestion
In order to receive historic messages from a `Mailserver`, a node MUST trust the selected `Mailserver`, that is allowed to send packets with the P2P Message code. By default, the node discards such packets.
```

File: docs/stable/4-whisper-mailserver.md [hunk 11]
```diff
@@ -67,7 +67,7 @@ The envelope MUST be encrypted with a symmetric key agreed between the requester
 
 Historic messages MUST be sent to a peer as a packet with a P2P Message code (`0x7f`) followed by an array of Whisper envelopes. It is incompatible with the original Whisper spec (EIP-627) because it allows only a single envelope, however, an array of envelopes is much more performant. In order to stay compatible with EIP-627, a peer receiving historic message MUST handle both cases.
 
-In order to receive historic messages from a `Mailserver`, a node MUST trust the selected `Mailserver`, that is allow to receive packets with the P2P Message code. By default, such packets are discarded.
+In order to receive historic messages from a `Mailserver`, a node MUST trust the selected `Mailserver`, that is allowed to receive packets with the P2P Message code. By default, the node discards such packets.
```
Comment (kdeme) [inline docs/stable/4-whisper-mailserver.md]: as in waku version

**Review by kdeme — APPROVED** [review]

File: docs/stable/11-waku-mailserver.md (hunk 10 above)
Comment (Samyoul) [reply docs/stable/11-waku-mailserver.md]: Hmm, good point, really well spotted.

**Review by decanus — APPROVED** [review]

**Pull request merged**